apps/aecore/src/aec_target.erl

```diff
@@ -14,7 +14,7 @@
 %% Returns {error | chain_too_short_to_recalculate_target} if initial height is a negative value
```

Comment:
Not that it's part of this PR, but why `|` here?

Reply:
That is just a the comment (and the comment is wrong, it returns `{error, chain_too_short...}`)